igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/logparser/ArtifactParser.groovy

```diff
@@ -24,18 +24,31 @@ import groovy.util.logging.Slf4j
 @Slf4j
 class ArtifactParser {
 
-    static def REGEXES = [ /Uploading artifact: https?:\/\/.+\/(.+\.(deb|rpm)).*/,
-                          /Successfully pushed (.+\.(deb|rpm)) to .*/ ]
+    static List<String> DEFAULT_REGEXES = [/Uploading artifact: https?:\/\/.+\/(.+\.(deb|rpm)).*/,
+                                           /Successfully pushed (.+\.(deb|rpm)) to .*/].toList()
 
+    /**
+     * Parse the build log using only DEFAULT_REGEXES.
+     */
     static List<GenericArtifact> getArtifactsFromLog(String buildLog) {
-        List<GenericArtifact> artifacts = new ArrayList<GenericArtifact>()
+        return getArtifactsFromLog(buildLog, Collections.EMPTY_LIST)
+    }
+
+    /**
+     * Parse the build log using DEFAULT_REGEXES plus the given ones.
+     */
+    static List<GenericArtifact> getArtifactsFromLog(String buildLog, Iterable<String> additionalRegexes) {
+        final List<GenericArtifact> artifacts = new ArrayList<GenericArtifact>()
+        final Set<String> regexes = new HashSet<>()
+        regexes.addAll(DEFAULT_REGEXES)
```

Comment:
I would prefer if injected regexes replaces the default ones. In some cases you would want to not use the standard regexes.